automations/js/src/label_pr.mjs

```diff
@@ -105,9 +109,15 @@ export const main = async (octokit, core) => {
   if (!getIsFullyLabeled(finalLabels.items)) {
     let attnLabel
     if (isTriaged) {
-      attnLabel = '🏷 status: label work required'
+      attnLabel = {
+        id: 'MDU6TGFiZWwzMDI5NTEyMjMw',
```

Comment:
They aren't likely to change unless we delete and recreate them. I would get them from the API but I worry about GitHub API's limits since we are making lots of calls (including GraphQL) from a single access token.